mm/include/z64save.h

```diff
@@ -386,6 +386,8 @@ typedef struct RandoSaveInfo {
     u32 finalSeed;
     u32 randoSaveOptions[RO_MAX]; // Type here may change in the future
     s8 foundDungeonKeys[9]; // Tracks the number of dungeon keys found, opposed to the number of keys in the inventory
+    s8 foundTriforcePieces;
+    s8 requiredTriforcePieces;
```

Comment:
This can be a u8 unless it being negative is ever a valid option.

Additionally, the amount required should be in a RANDO_OPTION
```suggestion
    u8 foundTriforcePieces;
```